src/utils/utils.ts

```diff
@@ -1,65 +1,90 @@
 export const cleanMarketsData = (
-  response: { id: string; marketname: any }[], distance: number
-) => {
+  response: { id: string; marketname: any }[], distance: number ) => {
```

Comment:
Just curious - why set the data type to any here?

Reply:
oof. yeah, I've corrected that now!